themes/vocabulary_theme/templates/blocks/recent-posts.html

```diff
@@ -1,20 +1,24 @@
 {% from "macros/posts.html" import render_post_summary %}
 
-<section class="recent-posts">
+<section class="recent-posts" aria-labelledby="recent-posts-title">
   <div class="container">
     <div class="level">
-      <h2 class="is-paddingless level-left">
+      <h2 id="recent-posts-title" class="is-paddingless level-left">
         {{ this.title }}
       </h2>
       <span class="level-right">
-        <a class="posts-link" href="/blog/entries">See all posts <i class="icon angle-right"></i></a>
+        <a class="posts-link" href="/blog/entries" aria-label="See all blog posts">
+          See all posts <i class="icon angle-right"></i>
+        </a>
       </span>
     </div>
-    <div class="columns">
+    <div class="columns" role="list">
```

Comment:
I'm not sure a list role here works better. If this needs to be a list then it should be encapsulated with a `ul` which already has that role. I don't love trying to turn a `div` into a semantic `ul` via a `role`